app/src/main/java/co/techmagic/hr/device/time_tracker/tracker_service/HrAppTimeTrackerService.kt

```diff
@@ -41,9 +41,7 @@ class HrAppTimeTrackerService : Service(), TimeTracker {
 
     private lateinit var wakeLock: PowerManager.WakeLock
 
-    private val publish: PublishSubject<TaskUpdate> = PublishSubject.create<TaskUpdate>().also {
-        it.onBackpressureDrop()
-    }
+    private val publish: PublishSubject<TaskUpdate> = PublishSubject.create()
```

Comment:
We have one more `PublishSubject` in the `TimeTrackerDataSource`. Probably should be fixed there as well. Could you check, please?